include/circt/Dialect/FIRRTL/FIRRTLAnnotationHelper.h

```diff
@@ -330,10 +330,7 @@ struct ApplyState {
   SmallVector<WiringProblem> wiringProblems;
 
   hw::InnerSymbolNamespace &getNamespace(FModuleLike module) {
-    auto &ptr = namespaces[module];
-    if (!ptr)
-      ptr = std::make_unique<hw::InnerSymbolNamespace>(module);
-    return *ptr;
+    return namespaces[module];
```

Comment:
I'm wondering why we had to hide that `InnerSymbolNamespace` away behind a `unique_ptr` in the first place :thinking: :sweat_smile: